models/abacus.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+
+const abacusSchema=new mongoose.Schema({
+
+    name: {type: String, required: true},
+    description: {type: String,required: true},
+    startDate: {type: Date, required: true},
+    endDate: {type: Date, required: true},
+    eventType: {type: String, required: true},
+    participants: [{name:{type: String},scholarID:{type: Number}}],
```

Comment:
Build a different schema to store responses and store the reference to those responses here.